includes/bootstrap/SWSFeatureContext.php

```diff
@@ -300,6 +300,26 @@ public function iClickOnTheElementWithCSSSelector($cssSelector) {
   }
 
   /**
+   * Press the element with the provided CSS Selector
+   *
+   * @When /^I press the element with css selector "([^"]*)"$/
+   */
+  public function iPressTheElementWithCSSSelector($cssSelector) {
+    $session = $this->getSession();
+    $xpath = $session->getSelectorsHandler()->selectorToXpath('css', $cssSelector);
+    $element = $session->getPage()->find(
+        'xpath',
+        $xpath
+    );
+
+    if (null === $element) {
+        throw new \InvalidArgumentException(sprintf('Could not evaluate CSS Selector: "%s"', $cssSelector));
+    }
+
+    $element->press();
```

Comment:
Press is specific to buttons. 

This step is for cases where there are multiple buttons with the same label or for when you are unsure what the label or name will be but you know where the button is going to be.